src/main/kotlin/app/Main.kt

```diff
@@ -0,0 +1,17 @@
+package app
+
+import app.ui.profilePreview.View.ProfilePreview
+import app.ui.welcomeScreen.*
+import tornadofx.*
+import java.util.*
+
+class MyApp : App(WelcomeScreen::class) {
+    companion object {
+        val Colors = ResourceBundle.getBundle("Colors")
```

Comment:
Can you see if this will work in it's own separate object? (specifically object, not class)

Reply:
yes it does work. changed it